### ISSUE TITLE
Active links need an `aria-current` attribute for accessibility

### ISSUE THREAD

**Issue by diondiondion** [codeplant/simple-navigation#234]
Active links generated by this gem are currently only highlighted with a class. This can only be used to _visually_ highlight such links, but it doesn't do anything to convey the selection state to people with visually impairments who rely on assistive technology like screen readers.

The fix should be easy: The (most deeply nested) active link should have the attribute `aria-current="page"`.

While this can be achieved with a custom renderer, it should be added to the library, as any pages using it as-is would find themselves in violation of e.g. the WCAG Success Criterion [2.4.8 Location](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.1&showtechniques=248#location) ("Information about the user's location within a set of web pages is available.")

Thanks for looking into this!

Further reading:
- [`aria-current` specification](https://www.w3.org/TR/wai-aria/#aria-current)
- WCAG 2.2 Techniques: [Indicating current location within navigation bars](https://www.w3.org/WAI/WCAG22/Techniques/general/G128)

**Comment by andi** [comment]
Thanks for the heads up. Would you have time to add an PR?

**Comment by diondiondion** [comment]
Thanks for your reply, unfortunately my Ruby/Rails knowledge is very limited, so I'm not confident I can create a PR myself.
